guides/doc-Administering_Red_Hat_Satellite/master.adoc

```diff
@@ -43,6 +43,8 @@ include::topics/Configuring_Identity_Management.adoc[]
 
 include::topics/Monitoring_Resources.adoc[]
 
+include::common/assembly_using-satellite-ansible-content-collections.adoc[]
```

Comment:
The assembly looks great now. How about moving it to other Ansible related content? For example, to line 21 after `include::topics/adding_rhel_system_roles.adoc[]`?

Reply:
Done